packages/gated-content/src/environments.ts

```diff
@@ -3,18 +3,21 @@ import { EthereumAddress } from '@lens-protocol/shared-kernel';
 import { SupportedChains } from './conditions/types';
 
 export type EnvironmentConfig = {
+  name: string;
```

Comment:
Instead of free string type we can define the values right? something like this

`'prod' | 'stg' | 'sandbox'`